src/api/search.js

```diff
@@ -72,21 +72,17 @@ class Search {
 
 
   static async autocomplete(searchOptions={}) {
-    // NOTE: I'm not sure how caching works with getCurrentPosition,
-    // but this could be problematic here if needs to compute each time.
-    // We could add a cache: true flag to retreive previous value
-    if (!searchOptions.near) {
-      const { latitude, longitude } = await Navigator.getCurrentPosition();
-      searchOptions.near = { latitude, longitude };
-    }
+    // if near is not provided, server will use geoIP as fallback
 
     let {
       query,
       near,
       limit,
     } = searchOptions;
 
-    near = `${near.latitude},${near.longitude}`;
+    if (near?.latitude && near?.longitude) {
```

Comment:
We we using this operator anywhere else?

Reply:
Want to make sure this is getting compiled properly

Reply:
ah good call, we arent using it anywhere in js sdk, but just ran the build and looks ok
```
if (((_near = near) === null || _near === void 0 ? void 0 : _near.latitude) && ((_near2 = near) === null || _near2 === void 0 ? void 0 : _near2.longitude)) {
                  near = "".concat(near.latitude, ",").concat(near.longitude);
                }

                params = {
                  query: query,
                  near: near,
                  limit: limit
                };
```
getting expanded to the null checks